homeassistant/components/iaqualink/binary_sensor.py

```diff
@@ -26,7 +26,10 @@ async def async_setup_entry(
 ) -> None:
     """Set up discovered binary sensors."""
     async_add_entities(
-        (HassAqualinkBinarySensor(dev) for dev in hass.data[AQUALINK_DOMAIN][DOMAIN]),
+        (
+            HassAqualinkBinarySensor(dev)
+            for dev in hass.data[AQUALINK_DOMAIN][BINARY_SENSOR_DOMAIN]
+        ),
```

Comment:
```suggestion
        HassAqualinkBinarySensor(dev)
        for dev in hass.data[AQUALINK_DOMAIN][BINARY_SENSOR_DOMAIN]
```

Reply:
Probably missing the comma as `add_entities` can take a generator.
Anyway, we can skip that as it's not related to the PR really.